src/OpenRiaServices.Server.EntityFrameworkCore/Test/DbContextModel/Scenarios/ComplexTypes.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+using Microsoft.EntityFrameworkCore;
+
+#nullable enable
+
+#if NET8_0_OR_GREATER
+
+namespace EFCoreModels.Scenarios.ComplexTypes
+{
+
+    public class Address
+    {
+        public required string AddressLine { get; set; }
+        public required string City { get; set; }
+    }
+
+    public class ContactInfo
+    {
+        public required Address Address { get; set; }
+
+        public required string HomePhone { get; set; }
+
+        public int PossibleId { get; set; }
+    }
+
+    public class Employee
+    {
+        public int EmployeeId { get; set; }
+        public required ContactInfo ContactInfo { get; set; }
+    }
+
+    public class ComplexTypesDbContext : DbContext
+    {
+        public ComplexTypesDbContext()
+        {
+        }
+
+        public ComplexTypesDbContext(DbContextOptions<ComplexTypesDbContext> options)
+            : base(options)
+        {
+        }
+
+        public DbSet<Employee> Employees { get; set; }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            // add concurrencymode = Fixed on all "Owned types" ??
+
+            modelBuilder.Owned<Address>();
+            //modelBuilder.Owned<ContactInfo>();
```

Comment:
This "owned"-part should be probably be removed

Reply:
removed